packages/cli/src/bin/ceramic.ts

```diff
@@ -24,6 +24,8 @@ program
     .option('--max-healthy-memory <decimal>', 'Fraction of total memory usage considered healthy. Defaults to 0.7')
     .option('--cors-allowed-origins <list>', 'Space-separated list of strings and/or regex expressions to set for Access-Control-Allow-Origin . Defaults to all: "*"')
     .option('--disable-anchors', 'Allows Ceramic Daemon to start up without a configured anchor service. Any anchor requests made will fail.')
+    .option('--no-restore-streams', 'Disable loading pinned stream state into memory at startup')
```

Comment:
I wonder if we need this option, or if we should just disable the stream restoration behavior whenever we're running in gateway mode?  Now that we don't sync the streams at startup as part of this, the "restore" functionality is just to poll the CAS for pending anchors that were created by this node, but if the node is a gateway it shouldn't have created any anchors. If you had a node that was writeable and then got restarted as a gateway you could miss some anchor commits, but that seems like a pretty niche case that maybe it's fine if we don't support anyway.

In retrospect, I wonder if I should get rid of the "--disble-anchors" flag also and just make that the behavior of the gateway mode.  The gateway should never have need of a CAS.

Reply:
oh ok, yeah i thought i remembered the sync streams at start changed, agree that this makes senes to fold into gateway mode, dont have any concerns with that. Looks like this covers all that already, right? - https://github.com/ceramicnetwork/js-ceramic/pull/1513/files